JellyfishClientDemo/Debug.xcconfig

```diff
@@ -0,0 +1 @@
+jellyfish_url = ws:/$()/192.168.0.164:4000/socket/peer/websocket
```

Comment:
it should be gitignored, that's the whole point of this, so I can modify address without changing anything in the git